### PR TITLE
Init nuxt & packages

### PR DESCRIPTION
**Pull request by themari0pl**
Created base package containing nuxt, daisyui, tailwind and pinia. Modified existing config files to allow those packages to work.

### CODE REVIEW

File: components/Footer/index.vue
Comment (lobarzanek) [inline components/Footer/index.vue:1]: Remove this component with the parent folder. We don't plan to use it in this project

File: app.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <div>
+    <TopNav/>
+    <NuxtPage/>
+    <Footer/>
```
Comment (lobarzanek) [inline app.vue]: Remove Footer component. We don't plan to use it in this project

File: app.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <div>
+    <TopNav/>
```
Comment (lobarzanek) [inline app.vue]: Change TopNav to SideBar component

File: components/Top/Nav.vue
Comment (lobarzanek) [inline components/Top/Nav.vue:1]: Remove this component and create a SideBar file instead. You can also rename it but clear it, leaving the component empty
You can use below path:
`components/Side/Bar.vue`

File: package.json
Comment (lobarzanek) [inline package.json:1]: Add `json-server` to dependencies and create `db.json` file in main folder
https://www.npmjs.com/package/json-server

After it add 
`"json": "json-server --watch db.json --port 8000"`
to scripts section

  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
     "json": "json-server --watch db.json --port 8000",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },

File: pages/about.vue
Comment (lobarzanek) [inline pages/about.vue:1]: Remove this page, we don't plan to use it in this project

**Review by lobarzanek — CHANGES_REQUESTED** [review]

File: pages/about.vue
Comment (themari0pl) [reply pages/about.vue:1]: done.

File: components/Side/Bar.vue [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<template>
+    <div>
+      <SideBar/>
+      <NuxtPage/>
```
Comment (lobarzanek) [inline components/Side/Bar.vue]: delete
`<SideBar/>`
`<NuxtPage/>`

File: nuxt.config.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+import { resolve } from "path";
+export default defineNuxtConfig({
+  devtools: { enabled: true },
+  css: ['~/assets/css/main.css'],
```
Comment (lobarzanek) [inline nuxt.config.ts]: Delete this line:
`css: ['~/assets/css/main.css'],`

You have this property currently in line 13
`css: ["~/assets/main.scss"]`

File: pages/about.vue
Comment (lobarzanek) [inline pages/about.vue:1]: Delete this page, We don't use it in this project

**Review by lobarzanek — APPROVED** [review]

**Review by kamilziemann — APPROVED** [review]

**Pull request merged**